optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/PartitionSolver.java

```diff
@@ -71,11 +71,7 @@ public String explainBestScore() {
 
     @Override
     public long getTimeMillisSpent() {
-        Long endingSystemTimeMillis = solverScope.getEndingSystemTimeMillis();
-        if (endingSystemTimeMillis == null) {
-            endingSystemTimeMillis = System.currentTimeMillis();
-        }
-        return endingSystemTimeMillis - solverScope.getStartingSystemTimeMillis();
+        throw new UnsupportedOperationException();
```

Comment:
Would be nice to tell a user why the operation is not supported to guide him towards a correct solution.

Reply:
The user never uses PartitionedSolver directly, it's a internal trick of PartitionedSearch to spawn multiple Solver. The other methods of PartitionedSolver that are inherited from Solver, also throw this exception. The difference is that this method will be deprecated with the new event.

Reply:
Normally I'd agree with this comment - in fact the PartitionedSearch code needs a review for such matters - and consistently change it - but separate issue, so I am going to merge that as is.

Reply:
@ge0ffrey would you please file an issue for the review so that we don't forget about it? Thanks